clientv3/doc.go

```diff
@@ -71,14 +71,14 @@
 //			// ctx is canceled by another routine
 //		} else if err == context.DeadlineExceeded {
 //			// ctx is attached with a deadline and it exceeded
+//		} else if err == rpctypes.ErrEmptyKey {
+//			// client-side error: key argument is empty
```

Comment:
Nit, Not sure if @gyuho meant to have a similar doc update as his comment here, https://github.com/coreos/etcd/blob/master/clientv3/example_kv_test.go#L69 
'client-side error: etcdserver: key is not provided' But preferably we should keep it similar Thanks!